client/src/modules/assets/modals/search.modal.js

```diff
@@ -44,18 +39,31 @@ function SearchStockAssignModalController(data, util, Store, Instance, Periods,
     displayValues.inventory_uuid = inventory.label;
   };
 
-  // custom filter entity_uuid - assign the value to the params object
-  vm.onSelectEntity = entity => {
-    vm.searchQueries.entity_uuid = entity.uuid;
-    displayValues.entity_uuid = entity.display_name;
+  // custom filter group_uuid - assign the value to the params object
+  vm.onSelectGroup = (group) => {
+    vm.searchQueries.group_uuid = group.uuid;
+    displayValues.group_uuid = group.name;
   };
 
-  // assign already defined custom filters to searchQueries object
-  vm.searchQueries = util.maskObjectFromKeys(data, searchQueryOptions);
+  // Save the entry dates
+  vm.onEntryDate = (dateFrom, dateTo) => {
+    vm.searchQueries.entry_date_from = dateFrom;
+    displayValues.entry_date_from = dateFrom;
+    // if (angular.isDefined(dateFrom)) {
```

Comment:
If these comments aren't necessary, go ahead and remove them.

Reply:
Removed